epregressions/tests/test_runtests.py

```diff
@@ -1880,6 +1888,81 @@ def test_external_interface_is_executed(self):
         self.assertEqual(EndErrSummary.STATUS_SUCCESS, results_for_file.summary_result.simulation_status_case1)
         self.assertEqual(EndErrSummary.STATUS_SUCCESS, results_for_file.summary_result.simulation_status_case2)
 
+    def test_ashrae_205_files_execute(self):
+        base = CMakeCacheMakeFileBuildDirectory()
+        self.establish_build_folder(
+            self.temp_base_build_dir,
+            self.temp_base_source_dir,
+            {
+                "config": {
+                    "run_time_string": "01hr 20min  0.17sec",
+                    "num_warnings": 1,
+                    "num_severe": 0,
+                    "end_state": "success",
+                    "eso_results": "base",
+                    "extra_data": ':ASHRAE205'
+                }
+            }
+        )
+        base.set_build_directory(self.temp_base_build_dir)
+
+        mod = CMakeCacheMakeFileBuildDirectory()
+        self.establish_build_folder(
+            self.temp_mod_build_dir,
+            self.temp_mod_source_dir,
+            {
+                "config": {
+                    "run_time_string": "00hr 10min  0.17sec",
+                    "num_warnings": 2,
+                    "num_severe": 1,
+                    "end_state": "success",
+                    "eso_results": "base",
+                    "extra_data": ':ASHRAE205'
+                }
+            }
+        )
+        mod.set_build_directory(self.temp_mod_build_dir)
+
+        entries = [TestEntry('my_file.idf', 'my_weather')]
+        config = TestRunConfiguration(
+            force_run_type=ForceRunType.NONE,
+            single_test_run=False,
+            num_threads=1,
+            report_freq=ReportingFreq.HOURLY,
+            build_a=base,
+            build_b=mod,
+            force_output_sql=ForceOutputSQL.SIMPLE,
+            force_output_sql_unitconv=ForceOutputSQLUnitConversion.NONE
```

Comment:
Also hijacking this unit test to exercise the force SQL functionality which gets unit test coverage back up to 100%.